.github/workflows/CICD.yml

```diff
@@ -210,13 +213,13 @@ jobs:
       with:
           command: install
           args: cargo-deb
-      if: ${{ contains(matrix.job.target, 'musl') }}
+      if: matrix.job.target == 'i686-unknown-linux-musl' || matrix.job.target == 'x86_64-unknown-linux-musl'
```

Comment:
Removed templating per docs

> When you use expressions in an if conditional, you may omit the expression syntax (${{ }}) because GitHub automatically evaluates the if conditional as an expression.